docs/references/kontena-yml.md

```diff
@@ -74,7 +79,7 @@ instances: 1
 
 #### stateful
 
-Mark service as stateful (default: false). Kontena will create and automatically mount a data volume container for the service. This option also instructs the scheduler to bind the service instance to the scheduled host so that the volume can be mapped when the service is updated.
+Mark service as stateful (default: false). Kontena will create and automatically mount a data volume container for the service. This option binds each service instance to the host node that is first scheduled on to, so that the same volumes can be mapped whenever the service is updated. This option also instructs the scheduler to not make any automatic scheduling decisions for the service after initial deployment is done.
```

Comment:
This reminds me that the `using-kontena/nodes.md` `Decomissioning nodes` should have some mention of how `kontena node rm` will cause stateful service instances to be re-deployed to new nodes and lose their state.